CHANGELOG-1.x.md

```diff
@@ -1,7 +1,7 @@
 # 1.X (TBD)
 
 - [06257ba](https://github.com/patternfly/patternfly-elements/commit/06257ba7edc3ca70ddfd0dd6c71e600540ac692b) fix: pfe-accordion accessibility issues
-
+- [fb11fba](https://github.com/patternfly/patternfly-elements/commit/fb11fba1932cde8f6f6d1632000233f0394d9c1c) fix: pfe-codeblock - update theme colours for accessibility
```

Comment:
```suggestion
- [](https://github.com/patternfly/patternfly-elements/commit/) fix: pfe-codeblock - update theme colours for accessibility
```

It's okay to leave this blank - we won't know the commit hash until it's merged with master